web/src/components/l10n/InstallerLocaleSwitcher.jsx

```diff
@@ -40,10 +42,32 @@ export default function InstallerLocaleSwitcher() {
   const options = Object.keys(languages).sort()
     .map(id => <FormSelectOption key={id} value={id} label={languages[id]} />);
 
+  // TRANSLATORS: help text for the language selector in the sidebar,
+  // %s will be replaced by the "Localization" page link
+  const [msg1, msg2] = _("The language used by the installer. The language \
+for the installed system can be set in the %s page.").split("%s");
+
+  // "hide" is a function which closes the popover
+  const description = (hide) => (
```

Comment:
Do you really need the hide callback here? I mean, when the user clicks on "Localization" I expect the installer navigating to Localization... and closing the Sidebar. Still the popover visible? If so, something to fix.

Reply:
Without the callback the link closes the sidebar and navigates to the localization page but the popover is still open.